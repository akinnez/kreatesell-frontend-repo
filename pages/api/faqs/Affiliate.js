export const questions = [
    {
        question: "What is the KreateSell Affiliate Program? ", 
        answer: `The Kreatesell affiliate program is a simple promotional system where you can get unique referral links of Kreators' products and promote them on social media or your personal blog/websites to reach a massive market and bring in more sales to the Kreators as an affiliate, you earn a preset commission per sales made from the buyers you send through your affiliate link.
        `
    },
    {
        question: "How do I become an Affiliate?", 
        answer: `To become an affiliate on Kreatesell, create an account on the platform as a user, verify the account and fill in your store details. After that, click on the “Find product” menu on the navigation bar. There would be a prompt to confirm if you would like to be an affiliate. 
        <br/><br/>
        After confirming, you would have access to the marketplace to see listed products. You can now browse through the products, find the ones you are interested in, then you can request access. After you’ve been approved by the Kreator, you can get your unique link to promote the product(s) and start earning commissions for every sale you make. Yes! You are free to start promoting immediately.
        <br/><br/>
        After five successful sales, you would no longer need to request access to be an affiliate. You can get unique links for any product you want. 
        `
    },
    {
        question: "How do I make money as an Affiliate on Kreatesell?", 
        answer: `You earn by requesting to be an affiliate for a product listed in the Kreatesell marketplace. Once you have been approved, you would receive a unique referral link and can start promoting the product through any of your marketing channels.
        <br/><br/>
        Once you've made five sales through any of your affiliate links on Kreatesell, you would no longer need to request approval to be an affiliate for a product. You can instantly get as many links as you want and earn massively.
        `
    },
    {
        question: "How much money can I make as a Kreatesell Affiliate?", 
        answer: `The money you make as an affiliate on Kreatesell is dependent on your marketing efforts and audience size. You can make as much as you want per time.
        <br/><br/>
        Let's talk about Samson.
        <br/><br/>
        Samson was always broke and known for borrowing until he decided to stop at all costs. He started to look for ways to make money online and discovered Kreatesell's Affiliate program. He signed up as a user on Kreatesell and started browsing through the products. He finally found the ones that offered great commissions to Affiliates.
        <br/><br/>
        After applying for them, he was approved and got his unique referral link for each product. He started promoting them on his Facebook, Instagram and Whatsapp status where he had a following and got so many people to buy - A total of 2,000!
        <br/><br/>
        He made commissions on every person who bought through his link and got paid instantly into his Kreatesell wallet and then, into his registered bank account or Mobile money account on our Affiliate payday - every Tuesday! 
        <br/><br/>
        So in less than 60 days, Samson Blew! He became a MULTI MILLIONAIRE in Naira!
        <br/><br/>
        Do you see? You can make as much as you want in as little time as you like as a Kreatesell Affiliate.
        `
    },
    {
        question: "Is the Kreatesell Affiliate program open to everyone?", 
        answer: `Yes, everyone can join our affiliate program. All you need is an active local bank account or mobile money account in a supported currency - Nigerian Naira (NGN), United States dollar (USD), Liberian Dollar (LRD), Ghanaian cedi (GHS), Kenya Shilling (KES), South African rand (ZAR), Tanzanian shilling (TZS), British pound sterling (GBP), Sierra Leonean Leone (SLL), Ugandan shilling (UGX), Malawian Kwacha (MWK), Gambian dalasi (GMD), West African CFA franc (XOF) or Central African CFA franc (XAF), to receive your commissions’ payment.
        `
    },
    {
        question: "Does Kreatesell teach Affiliates how to sell products?", 
        answer: `Yes! Kreatesell provides free training for affiliates via our Telegram channel. We also have a Facebook group for affiliates to interact with and learn from one another. If you would like more advanced training you can browse through the product marketplace as a user and purchase an affiliate marketing course offered by one or more of our Kreators.
        `
    },
    {
        question: "Can I join if I’m participating in another affiliate program?", 
        answer: `Yes, you can participate in thousands of affiliate programs at the same time. But, Kreatesell is the best affiliate program you can find. Join now and start earning massively on Kreatesell.
        `
    },
    {
        question: "Can I participate from any country?", 
        answer: `Yes, you can participate from everywhere and anywhere you are. Once you have a means for receiving payments in Nigerian Naira (NGN), United States dollar (USD), Liberian Dollar (LRD), Ghanaian cedi (GHS), Kenya Shilling (KES), South African rand (ZAR), Tanzanian shilling (TZS), British pound sterling (GBP), Sierra Leonean Leone (SLL), Ugandan shilling (UGX), Malawian Kwacha (MWK), Gambian dalasi (GMD), West African CFA franc (XOF) or Central African CFA franc (XAF), you are good to go.`
    },
    {
        question: "When do I get paid?", 
        answer: `Total Affiliate commissions are paid every Tuesday.
        `
    },
    {
        question: "How do I receive my payment?", 
        answer: `Payments are made into your registered payment choice. This could be your local bank account, mobile money wallet, or Paypal.`
    },
    {
        question: "How can I track the success of my promotion?", 
        answer: `You can track data of how your promotion link is faring through your affiliate dashboard or Product Marketplace.`
    },
    {
        question: "What sort of promotional materials do you provide to Affiliates?", 
        answer: `Kreators may provide Content, Videos, Images and Email swipes as part of setting up their digital product, for affiliates to use when promoting their products.`
    },
    {
        question: "Do I need to pay to become an Affiliate?", 
        answer: `No, it is totally free to create an account and become an affiliate on Kreatesell. <a href="/signup">Go here to create an account</a>.
        `
    },
    {
        question: "Must I be approved before I can be an Affiliate for a product on Kreatesell?", 
        answer: `You have to apply and be approved by a Kreator to be their affiliate and get a unique link for marketing their product. However, if you have successfully made five sales, you would be allowed direct access to get unique links and promote any product without having to go through the approval process from the Kreator.
        `
    },
    {
        question: "How do I receive a payment if I don't have an account to receive foreign currencies from other countries?", 
        answer: `Kreatesell has taken care of the foreign currencies barrier you might face in getting paid. No matter the currency in which a customer buys using your link, you would get the corresponding commission in the local currency that your registered bank account, mobile money account or PayPal supports.`
    }
  ]