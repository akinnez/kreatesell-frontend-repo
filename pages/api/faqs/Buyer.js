export const questions = [
    {
        question: "If the product bought is not genuine, can I get a refund?", 
        answer:"Yes, you can request a refund within 30 days of purchase if the product has a refund policy attached."
    },
    {
        question: "How do I request a refund if I donâ€™t want a product I bought anymore?", 
        answer:"Kreatesell is an easy-to-use, cross-border education technology (edtech) SaaS platform that enables content kreators and digital entrepreneurs to upload their digital products for the fastest and easiest sales. With KreateSell, you can convert your audience into buying customers without going through the stress of learning many sales funnels that may be complex to use. Also, you instantly get paid from anywhere in the world conveniently. See more about how it works. Kreatesell is solving the major problems that Kreators face while selling online including having no means to receive payment in foreign currency. With Kreatesell, this is no longer a barrier because buyers can now pay in their local currency and Kreators can receive the money seamlessly in their local currencies without going through the stress of trying to exchange currencies using third-party platforms. On Kreatesell, you have access to a large number of affiliates ready to promote your product to potential buyers with a preset commission fee of your choice. Also, you do not have to go through the stress of manual delivery of your digital product, your delivery process is automated for you. Your customer receives the product immediately after payment is confirmed."
    },
    {
        question: "How safe is it to make payment with my card on your website?", 
        answer:"Kreatesell is an easy-to-use, cross-border education technology (edtech) SaaS platform that enables content kreators and digital entrepreneurs to upload their digital products for the fastest and easiest sales. With KreateSell, you can convert your audience into buying customers without going through the stress of learning many sales funnels that may be complex to use. Also, you instantly get paid from anywhere in the world conveniently. See more about how it works. Kreatesell is solving the major problems that Kreators face while selling online including having no means to receive payment in foreign currency. With Kreatesell, this is no longer a barrier because buyers can now pay in their local currency and Kreators can receive the money seamlessly in their local currencies without going through the stress of trying to exchange currencies using third-party platforms. On Kreatesell, you have access to a large number of affiliates ready to promote your product to potential buyers with a preset commission fee of your choice. Also, you do not have to go through the stress of manual delivery of your digital product, your delivery process is automated for you. Your customer receives the product immediately after payment is confirmed."
    },
    {
        question: "How does the bank transfer payment option work?", 
        answer:"Kreatesell is an easy-to-use, cross-border education technology (edtech) SaaS platform that enables content kreators and digital entrepreneurs to upload their digital products for the fastest and easiest sales. With KreateSell, you can convert your audience into buying customers without going through the stress of learning many sales funnels that may be complex to use. Also, you instantly get paid from anywhere in the world conveniently. See more about how it works. Kreatesell is solving the major problems that Kreators face while selling online including having no means to receive payment in foreign currency. With Kreatesell, this is no longer a barrier because buyers can now pay in their local currency and Kreators can receive the money seamlessly in their local currencies without going through the stress of trying to exchange currencies using third-party platforms. On Kreatesell, you have access to a large number of affiliates ready to promote your product to potential buyers with a preset commission fee of your choice. Also, you do not have to go through the stress of manual delivery of your digital product, your delivery process is automated for you. Your customer receives the product immediately after payment is confirmed."
    },
    {
        question: "How am I sure about the quality of products on your platform?", 
        answer:"Kreatesell is an easy-to-use, cross-border education technology (edtech) SaaS platform that enables content kreators and digital entrepreneurs to upload their digital products for the fastest and easiest sales. With KreateSell, you can convert your audience into buying customers without going through the stress of learning many sales funnels that may be complex to use. Also, you instantly get paid from anywhere in the world conveniently. See more about how it works. Kreatesell is solving the major problems that Kreators face while selling online including having no means to receive payment in foreign currency. With Kreatesell, this is no longer a barrier because buyers can now pay in their local currency and Kreators can receive the money seamlessly in their local currencies without going through the stress of trying to exchange currencies using third-party platforms. On Kreatesell, you have access to a large number of affiliates ready to promote your product to potential buyers with a preset commission fee of your choice. Also, you do not have to go through the stress of manual delivery of your digital product, your delivery process is automated for you. Your customer receives the product immediately after payment is confirmed."
    },
    {
        question: "How do I download a product I paid for?", 
        answer:"Kreatesell is an easy-to-use, cross-border education technology (edtech) SaaS platform that enables content kreators and digital entrepreneurs to upload their digital products for the fastest and easiest sales. With KreateSell, you can convert your audience into buying customers without going through the stress of learning many sales funnels that may be complex to use. Also, you instantly get paid from anywhere in the world conveniently. See more about how it works. Kreatesell is solving the major problems that Kreators face while selling online including having no means to receive payment in foreign currency. With Kreatesell, this is no longer a barrier because buyers can now pay in their local currency and Kreators can receive the money seamlessly in their local currencies without going through the stress of trying to exchange currencies using third-party platforms. On Kreatesell, you have access to a large number of affiliates ready to promote your product to potential buyers with a preset commission fee of your choice. Also, you do not have to go through the stress of manual delivery of your digital product, your delivery process is automated for you. Your customer receives the product immediately after payment is confirmed."
    },
    {
        question: "I just bought a product on Kreatesell, can I also sell a product I own?", 
        answer:"Kreatesell is an easy-to-use, cross-border education technology (edtech) SaaS platform that enables content kreators and digital entrepreneurs to upload their digital products for the fastest and easiest sales. With KreateSell, you can convert your audience into buying customers without going through the stress of learning many sales funnels that may be complex to use. Also, you instantly get paid from anywhere in the world conveniently. See more about how it works. Kreatesell is solving the major problems that Kreators face while selling online including having no means to receive payment in foreign currency. With Kreatesell, this is no longer a barrier because buyers can now pay in their local currency and Kreators can receive the money seamlessly in their local currencies without going through the stress of trying to exchange currencies using third-party platforms. On Kreatesell, you have access to a large number of affiliates ready to promote your product to potential buyers with a preset commission fee of your choice. Also, you do not have to go through the stress of manual delivery of your digital product, your delivery process is automated for you. Your customer receives the product immediately after payment is confirmed."
    },
]