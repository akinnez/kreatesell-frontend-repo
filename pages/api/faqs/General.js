export const questions = [
    {
        question: "Do I need to install anything? ", 
        answer:"No! This is a web-based platform. With Keatesell you do not need to go through the stress of installing software, tools or applications. Everything has been done for you. So, with just a few clicks, you can start making massive sales from anywhere at any time."
    },
    {
        question: "If I have questions that are not covered here, is there someone I can talk to?", 
        answer: `Yes! We have a great support team that is ready to help you 24/7. As a user on Kreatesell, you can contact us about your concerns via our live chat, Whatsapp and email support channel: <a href="mailto:support@kreatesell.com">support@kreatesell.com</a>. If you are yet to create an account on Kreatesell, you can make enquiries via <a href="hello@kreatesell.com">hello@kreatesell.com</a>. All our social media handles are also open for enquiries, you can send us a direct message and get a prompt response from us.
        `
    }
]