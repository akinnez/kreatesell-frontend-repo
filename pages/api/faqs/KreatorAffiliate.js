export const questions = [
    {
        question: "How long does it take to receive my money?", 
        answer: `On Kreatesell, we provide payout and settlement at due intervals of 24 hours to 10 days depending on your currency and country. Check out your wait time <a href="">here</a>.
        <br/><br/>
        If payment is not received by the set date, you can report the issue to us at <a href="mailto: payment@kreatesell.com">payment@kreatesell.com</a>
        `
    },
    {
        question: "How do I know if I have made sales?", 
        answer: `You get email notifications from Kreatesell for every successful sales transaction.
        <br/><br/>
        You can also monitor how much sales have been made on your digital product (for Kreator) or the product you are promoting (for Affiliate) through the transaction history on your dashboard.
        `
    },
    {
        question: "Can I sign up with my social media account?", 
        answer: `Yes! You can sign up on Kreatesell with your social media account. After clicking "Get Started Free" or "sign up" to create an account, you can click on the Facebook button to sign up with your Facebook social media account. Also, you can click on the Google button to sign up with your Gmail account.
        `
    },
    {
        question: "Are my login details safe?", 
        answer: `Yes, Kreatesell provides maximum security on our website. So, your login information is safe on the site. All passwords entered into our website are encrypted so, they are not visible to anyone, including us. We also have the option for you to activate Two Factor Authentication (2FA). If you have issues with logging in to your account, send us an email at <a href=""mailto: business@kreatesell.com">business@kreatesell.com</a> with your registered email address.
        `
    },
    {
        question: "Can I be both an Affiliate and a Kreator on Kreatesell at the same time?", 
        answer: `Yes! Signing up as a user on Kreatesell allows you to be either a Kreator, an affiliate or both.
        <br/><br/>
        If you are an existing Kreator and are interested in being an affiliate for a product listed in the Kreatesell marketplace, all you need to do is request to be an affiliate, once you are approved, you get your referral link and you can start promoting.
        <br/><br/>
        If you are an affiliate already but want to start selling as a Kreator, just click on Add product, follow the prompts, fill in the required details and you’re good to go.
        `
    },
    {
        question: "What is the minimum withdrawal amount?", 
        answer: `The payout for Nigerian Kreators is automatic. But for Kreators outside Nigeria, payouts are made immediately you click the withdraw button. You receive the total balance in your wallet at once. Before you can withdraw, you are required to have a minimum balance as follows:
        <br/><br/>
        Nigerian Naira (NGN): NGN 50
        Ghana Cedi (GHS): GHS 10
        Kenyan Shilling (KES): KES 100
        Tanzanian Shilling (TZS): TZS 3500
        Ugandan Shilling (UGX): UGX 12, 000
        South African Rands (ZAR): ZAR 12
        Central African CFA franc (XAF): 600 XAF
        West African CFA franc (XOF): 600 XOF
        US Dollars (USD): USD 0 - Free (via PayPal)
        British Pounds (GBP): GBP 0 - Free (via PayPal)
        This serves as the withdrawal fees for taking funds out of your wallet.
        <br/><br/>
        However, as an affiliate, you have to make successful sales and accumulate commissions to a threshold amount of N5,000 or $20 for you to get your first payment. Payouts are made every Tuesday.
        `
    },
    {
        question: "Why am I yet to receive my settlement/payout?", 
        answer: `On Kreatesell, we provide payout and settlement at due intervals of 24 hours to 10 days depending on your currency and country. Check out your wait time <a href="">here</a>.
        <br/><br/>
        If payment is not received by the set date, you can report the issue to us at "<a href="mailto=payment@kreatesell.com">payment@kreatesell.com</a>.
        `
    },
    {
        question: "Can I cancel my Kreatesell account if I don't want to use it anymore?", 
        answer: `Yes! To delete your Kreatesell account, click on the HELP menu and fill out the form. You would be promptly attended to by our customer success support team who are always ready to help.
        `
    },
    {
        question: "If I delete my Kreatesell account, would I lose my data?", 
        answer: `Considering that Kreatesell is a software as a service (SaaS) platform hosted on the web, all data on our platform is stored in the cloud. So, deleting your account means you delete all the information on the cloud and won’t be able to access it. So, we advise that you keep your account in case you would love to access your data in the future.`
    },
    {
        question: "Who owns my information/data/content on Kreatesell?", 
        answer: `Every information you have put into Kreatesell totally belongs to you. Be rest assured that Kreatesell does not have any right to your content or information. Kreatesell is just a tool to help you make more massive sales in selling your digital product.`
    },
    
  ]