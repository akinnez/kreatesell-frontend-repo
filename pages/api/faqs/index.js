import {questions as AffiliateFaq} from "./Affiliate";
import {questions as BuyerFaq} from "./Buyer";
import {questions as GeneralFaq} from "./General";
import {questions as KreatorAffiliateFaq} from "./KreatorAffiliate";
import {questions as KreatorFaq} from "./Kreators";

export default function handler(req, res){
    let questions = {
        Affiliate: AffiliateFaq,
        Buyer: BuyerFaq,
        General: GeneralFaq,
        KreatorAffiliate: KreatorAffiliateFaq,
        Kreator: KreatorFaq
    }
    res.status(200).json({items: questions});
}