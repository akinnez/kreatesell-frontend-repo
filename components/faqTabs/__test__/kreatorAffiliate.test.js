import { questions } from "../../../pages/api/faqs/KreatorAffiliate";
import renderer from "react-test-renderer";
import KreatorAffiliate from "../KreatorAffiliate";

it("renders a snapshot of <Kreator/> tab", () => {
  const tab = renderer
    .create(<KreatorAffiliate questions={questions} />)
    .toJSON();
  expect(tab).toMatchSnapshot();
});
