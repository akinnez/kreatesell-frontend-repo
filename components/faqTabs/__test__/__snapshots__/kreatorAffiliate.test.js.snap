// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot of <Kreator/> tab 1`] = `
<div
  className="tabContainer"
>
  <h3
    className="title"
  >
    Kreator Affiliate
  </h3>
  <div
    className="ant-collapse ant-collapse-borderless ant-collapse-icon-position-right"
    role={null}
  >
    <div
      className="ant-collapse-item ant-collapse-item-active panelHeader"
    >
      <div
        aria-expanded={true}
        className="ant-collapse-header"
        onClick={[Function]}
        onKeyPress={[Function]}
        role="button"
        tabIndex={0}
      >
        <div>
          <span
            aria-label="right"
            className="anticon anticon-right ant-collapse-arrow"
            role="img"
          >
            <svg
              aria-hidden="true"
              data-icon="right"
              fill="currentColor"
              focusable="false"
              height="1em"
              style={
                Object {
                  "msTransform": "rotate(90deg)",
                  "transform": "rotate(90deg)",
                }
              }
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M765.7 486.8L314.9 134.7A7.97 7.97 0 00302 141v77.3c0 4.9 2.3 9.6 6.1 12.6l360 281.1-360 281.1c-3.9 3-6.1 7.7-6.1 12.6V883c0 6.7 7.7 10.4 12.9 6.3l450.8-352.1a31.96 31.96 0 000-50.4z"
              />
            </svg>
          </span>
        </div>
        How long does it take to receive my money?
      </div>
      <div
        className="ant-collapse-content ant-collapse-content-active"
        role={null}
      >
        <div
          className="ant-collapse-content-box"
        >
          <div
            className="answer"
          >
            On Kreatesell, we provide payout and settlement at due intervals of 24 hours to 10 days depending on your currency and country. Check out your wait time here.

        If payment is not received by the set date, you can report the issue to us at payment@kreatesell.com
        
          </div>
        </div>
      </div>
    </div>
    <div
      className="ant-collapse-item panelHeader"
    >
      <div
        aria-expanded={false}
        className="ant-collapse-header"
        onClick={[Function]}
        onKeyPress={[Function]}
        role="button"
        tabIndex={0}
      >
        <div>
          <span
            aria-label="right"
            className="anticon anticon-right ant-collapse-arrow"
            role="img"
          >
            <svg
              aria-hidden="true"
              data-icon="right"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M765.7 486.8L314.9 134.7A7.97 7.97 0 00302 141v77.3c0 4.9 2.3 9.6 6.1 12.6l360 281.1-360 281.1c-3.9 3-6.1 7.7-6.1 12.6V883c0 6.7 7.7 10.4 12.9 6.3l450.8-352.1a31.96 31.96 0 000-50.4z"
              />
            </svg>
          </span>
        </div>
        How do I know if I have made sales?
      </div>
    </div>
    <div
      className="ant-collapse-item panelHeader"
    >
      <div
        aria-expanded={false}
        className="ant-collapse-header"
        onClick={[Function]}
        onKeyPress={[Function]}
        role="button"
        tabIndex={0}
      >
        <div>
          <span
            aria-label="right"
            className="anticon anticon-right ant-collapse-arrow"
            role="img"
          >
            <svg
              aria-hidden="true"
              data-icon="right"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M765.7 486.8L314.9 134.7A7.97 7.97 0 00302 141v77.3c0 4.9 2.3 9.6 6.1 12.6l360 281.1-360 281.1c-3.9 3-6.1 7.7-6.1 12.6V883c0 6.7 7.7 10.4 12.9 6.3l450.8-352.1a31.96 31.96 0 000-50.4z"
              />
            </svg>
          </span>
        </div>
        Can I sign up with my social media account?
      </div>
    </div>
    <div
      className="ant-collapse-item panelHeader"
    >
      <div
        aria-expanded={false}
        className="ant-collapse-header"
        onClick={[Function]}
        onKeyPress={[Function]}
        role="button"
        tabIndex={0}
      >
        <div>
          <span
            aria-label="right"
            className="anticon anticon-right ant-collapse-arrow"
            role="img"
          >
            <svg
              aria-hidden="true"
              data-icon="right"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M765.7 486.8L314.9 134.7A7.97 7.97 0 00302 141v77.3c0 4.9 2.3 9.6 6.1 12.6l360 281.1-360 281.1c-3.9 3-6.1 7.7-6.1 12.6V883c0 6.7 7.7 10.4 12.9 6.3l450.8-352.1a31.96 31.96 0 000-50.4z"
              />
            </svg>
          </span>
        </div>
        Are my login details safe?
      </div>
    </div>
    <div
      className="ant-collapse-item panelHeader"
    >
      <div
        aria-expanded={false}
        className="ant-collapse-header"
        onClick={[Function]}
        onKeyPress={[Function]}
        role="button"
        tabIndex={0}
      >
        <div>
          <span
            aria-label="right"
            className="anticon anticon-right ant-collapse-arrow"
            role="img"
          >
            <svg
              aria-hidden="true"
              data-icon="right"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M765.7 486.8L314.9 134.7A7.97 7.97 0 00302 141v77.3c0 4.9 2.3 9.6 6.1 12.6l360 281.1-360 281.1c-3.9 3-6.1 7.7-6.1 12.6V883c0 6.7 7.7 10.4 12.9 6.3l450.8-352.1a31.96 31.96 0 000-50.4z"
              />
            </svg>
          </span>
        </div>
        Can I be both an Affiliate and a Kreator on Kreatesell at the same time?
      </div>
    </div>
    <div
      className="ant-collapse-item panelHeader"
    >
      <div
        aria-expanded={false}
        className="ant-collapse-header"
        onClick={[Function]}
        onKeyPress={[Function]}
        role="button"
        tabIndex={0}
      >
        <div>
          <span
            aria-label="right"
            className="anticon anticon-right ant-collapse-arrow"
            role="img"
          >
            <svg
              aria-hidden="true"
              data-icon="right"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M765.7 486.8L314.9 134.7A7.97 7.97 0 00302 141v77.3c0 4.9 2.3 9.6 6.1 12.6l360 281.1-360 281.1c-3.9 3-6.1 7.7-6.1 12.6V883c0 6.7 7.7 10.4 12.9 6.3l450.8-352.1a31.96 31.96 0 000-50.4z"
              />
            </svg>
          </span>
        </div>
        What is the minimum withdrawal amount?
      </div>
    </div>
    <div
      className="ant-collapse-item panelHeader"
    >
      <div
        aria-expanded={false}
        className="ant-collapse-header"
        onClick={[Function]}
        onKeyPress={[Function]}
        role="button"
        tabIndex={0}
      >
        <div>
          <span
            aria-label="right"
            className="anticon anticon-right ant-collapse-arrow"
            role="img"
          >
            <svg
              aria-hidden="true"
              data-icon="right"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M765.7 486.8L314.9 134.7A7.97 7.97 0 00302 141v77.3c0 4.9 2.3 9.6 6.1 12.6l360 281.1-360 281.1c-3.9 3-6.1 7.7-6.1 12.6V883c0 6.7 7.7 10.4 12.9 6.3l450.8-352.1a31.96 31.96 0 000-50.4z"
              />
            </svg>
          </span>
        </div>
        Why am I yet to receive my settlement/payout?
      </div>
    </div>
    <div
      className="ant-collapse-item panelHeader"
    >
      <div
        aria-expanded={false}
        className="ant-collapse-header"
        onClick={[Function]}
        onKeyPress={[Function]}
        role="button"
        tabIndex={0}
      >
        <div>
          <span
            aria-label="right"
            className="anticon anticon-right ant-collapse-arrow"
            role="img"
          >
            <svg
              aria-hidden="true"
              data-icon="right"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M765.7 486.8L314.9 134.7A7.97 7.97 0 00302 141v77.3c0 4.9 2.3 9.6 6.1 12.6l360 281.1-360 281.1c-3.9 3-6.1 7.7-6.1 12.6V883c0 6.7 7.7 10.4 12.9 6.3l450.8-352.1a31.96 31.96 0 000-50.4z"
              />
            </svg>
          </span>
        </div>
        Can I cancel my Kreatesell account if I donâ€™t want to use it anymore?
      </div>
    </div>
    <div
      className="ant-collapse-item panelHeader"
    >
      <div
        aria-expanded={false}
        className="ant-collapse-header"
        onClick={[Function]}
        onKeyPress={[Function]}
        role="button"
        tabIndex={0}
      >
        <div>
          <span
            aria-label="right"
            className="anticon anticon-right ant-collapse-arrow"
            role="img"
          >
            <svg
              aria-hidden="true"
              data-icon="right"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M765.7 486.8L314.9 134.7A7.97 7.97 0 00302 141v77.3c0 4.9 2.3 9.6 6.1 12.6l360 281.1-360 281.1c-3.9 3-6.1 7.7-6.1 12.6V883c0 6.7 7.7 10.4 12.9 6.3l450.8-352.1a31.96 31.96 0 000-50.4z"
              />
            </svg>
          </span>
        </div>
        If I delete my Kreatesell account, would I lose my data?
      </div>
    </div>
    <div
      className="ant-collapse-item panelHeader"
    >
      <div
        aria-expanded={false}
        className="ant-collapse-header"
        onClick={[Function]}
        onKeyPress={[Function]}
        role="button"
        tabIndex={0}
      >
        <div>
          <span
            aria-label="right"
            className="anticon anticon-right ant-collapse-arrow"
            role="img"
          >
            <svg
              aria-hidden="true"
              data-icon="right"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M765.7 486.8L314.9 134.7A7.97 7.97 0 00302 141v77.3c0 4.9 2.3 9.6 6.1 12.6l360 281.1-360 281.1c-3.9 3-6.1 7.7-6.1 12.6V883c0 6.7 7.7 10.4 12.9 6.3l450.8-352.1a31.96 31.96 0 000-50.4z"
              />
            </svg>
          </span>
        </div>
        Who owns my information/data/content on Kreatesell?
      </div>
    </div>
  </div>
</div>
`;
